lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -303,6 +311,13 @@ def set_list_position(new_position)
         end
 
         private
+
+          def swap_positions(item1, item2)
+            item1_position = item1.position
```

Comment:
Rather than this, could you look into just changing item1's position to item2's, then changing item2's position to `item1.position_was`? May as well take advantage of `ActiveModel::Dirty`. It may not work if `set_list_position` saves the change though.